workshop_sections/word2vec/README.md

```diff
@@ -1,120 +1,178 @@
 
-This walkthrough assumes that your working directory is that of this `README` file. 
+This walkthrough assumes that your working directory is that of this `README` file.
 
 ## Getting set up
 
-First make sure you have installed TensorFlow, numpy and nltk in your python environment.
+First make sure you have installed TensorFlow in your python environment.
 
-Previous workshop sections have instructions on how to install TensorFlow and numpy. To install nltk:
+Previous workshop sections have instructions on how to install TensorFlow.
+
+There are a number of options for how to train. Using preprocessed data from the cloud, locally, or preprocessing your own data from the corpus.
+
+
+### Option 1. Preprocessed files
+
+We have provided 3 preprocessed files, for your use directly in training. The preprocessing uses the [text8](http://mattmahoney.net/dc/textdata) which is a cleaned Wikipedia text dump. We then counted words in this corpus and created a "vocabulary file" containing the `2 ^ 15 - 1` most common words in the corpus in order of their occurance; `text8-index.tsv`. We then used this index to convert the corpus into integers, and split it 90-10 into a training data and test data corpus.
+
+#### 1a. Training from files in the Cloud.
+
+Since many TensorFlow ops use the `GFile` abstraction for reading from the filesystem, and `GFile` supports Google Cloud Storage (GCS), you can train without copying files locally. Simply train locally with the following command:
 
 ```
-pip install nltk
-python -c "import nltk; nltk.download('punkt')"
+python word2vec/task.py \
+    --output-path output/1 \
+    --vocab-file gs://tf-ml-workshop/word2vec/data/text8-vocab.tsv \
+    --train-data-file gs://tf-ml-workshop/word2vec/data/text8-train.pb2 \
+    --eval-data-file gs://tf-ml-workshop/word2vec/data/text8-eval.pb2
 ```
-(Note: if you are in using a Docker image this is already done)
 
-This will give you the necessary corpuses to use `nltk.word_tokenize`.
+#### 1b. Pulling files locally
 
-### Downloading the training data
+If you want to pull files ahead of time and train locally, simply copy them from the public GCS bucket.
 
-The training data can be found at [http://mattmahoney.net/dc/text8.zip](http://mattmahoney.net/dc/text8.zip) (25 MB compressed, ~100 MB uncompressed)
+```
+gsutil cp -r gs://tf-ml-workshop/word2vec/data/* ./
+```
 
-The rest of this tutorial assumes you have downloaded this data and unzipped it in this directory, you can do this with:
+Then use the local paths in your command, as follows:
 
-```sh
-curl http://mattmahoney.net/dc/text8.zip -o text8.zip
-unzip text8.zip
+```
+python word2vec/task.py \
+    --output-path output/1 \
+    --vocab-file text8-vocab.tsv \
+    --train-data-file text8-train.pb2 \
+    --eval-data-file text8-eval.pb2
 ```
 
-## Preproccessing Data
 
-All the following commands are run in a python interpreter. So start up a python intepreter with 
+### Option 2. Doing your own preprocessing
```

Comment:
I'm going to bump it up a level actually. No reason it should be in with the training code.